app/lib/matador/components/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./job-info";
 export * from "./job-result";
 export * from "./jobs-table";
 export * from "./error";
```

Comment:
```suggestion
export * from "./error";
```

This throws an error if I try to execute `npm run dev`